lib/XliffSelect.js

```diff
@@ -52,6 +53,14 @@ function wordCount(string) {
 var XliffSelect = function XliffSelect(settings) {
     if (!settings) return;
 
+    // Remember which files we have already read, so we don't have
+    // to read them again.
+    var fileNameCache = new ISet();
```

Comment:
Shouldn't we just use es6 Set instead of ilib one?

Reply:
Because the whole loctool is written with old Javascript (currently) and we cannot rely on `Set` being available in all versions of node we support.